### PR TITLE
misc: add tcllib

### PR DESCRIPTION
**Pull request by proppy**
- backported from https://github.com/conda-forge/tcllib-feedstock/
- needed by openlane

### CODE REVIEW

File: misc/tcllib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  license_family: BSD
+  license_file: license.terms
+  summary: 'Tcllib is a collection of utility modules for Tcl.'
+  doc_url: https://core.tcl-lang.org/tcllib/doc/trunk/embedded/md/toc.md
+
+extra:
+  recipe-maintainers:
+    - abloemert
+    - proppy
+    - xobs
```
Comment (proppy) [inline misc/tcllib/meta.yaml:52]: @xobs added you there, hope you don't mind?

Comment (xobs) [reply misc/tcllib/meta.yaml:52]: No worries, works for me!

**Comment by xobs** [comment]
LGTM

**Pull request merged**